### PR TITLE
nfs: do not run privileged nfs container

### PR DESCRIPTION
**Pull request by guits**
At the moment, we bindmount the dbus socket from the host, this requires
to run the container with --privileged.
Since we now run a dedicated dbus daemon inside the same container, we
can stop running privileged nfs-ganesha containers

Related ceph-container PR : ceph/ceph-container#1517

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1725254

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: roles/ceph-nfs/tasks/pre_requisite_container.yml [hunk 1]
```diff
@@ -26,16 +26,3 @@
         - item.item.copy_key | bool
   when: groups.get(mon_group_name, []) | length > 0
 
-- name: dbus related tasks
-  block:
-    - name: create dbus service file
-      copy:
-        src: "org.ganesha.nfsd.conf"
-        dest: /etc/dbus-1/system.d/org.ganesha.nfsd.conf
```
Comment (dsavineau) [inline roles/ceph-nfs/tasks/pre_requisite_container.yml]: I confirmed that this file exists in the nfs-ganesha package 2.7 (luminous), 2.8 (nautilus) and 3.0 (octopus)

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-update

File: roles/ceph-nfs/tasks/pre_requisite_container.yml (hunk 1 above)
Comment (guits) [reply roles/ceph-nfs/tasks/pre_requisite_container.yml]: ok maggle, thanks!

**Comment by tombarron** [comment]
I understand the motivation for this change, but if we do away with the dbus socket bind mount, how are other processes not running in this container (or a pod with it) supposed to communicate wiith ganesha to dynamically update the exports?

**Pull request merged**